bindings/java/src/org/sleuthkit/datamodel/WindowsAccountUtils.java

```diff
@@ -115,40 +122,128 @@
+	// These windows SID indicate well known windows accounts.
+	// Well known SIDs and account are handled slightly differently from the regular accounts:
+	//  - We can assume and fill in SID from given account name, and vice versa.
+	//  - We map account names in foreign languages (some known set) to english names, for these well known accounts. 
+	private static final Map<String, WellKnownSidInfo> SPECIAL_SIDS_MAP =  ImmutableMap.<String, WellKnownSidInfo>builder() 
+			.put("S-1-5-18", new WellKnownSidInfo(true, "S-1-5", NTAUTHORITY_REALM_NAME, "SYSTEM", "Local System Account"))
+			.put("S-1-5-19", new WellKnownSidInfo(true, "S-1-5", NTAUTHORITY_REALM_NAME, "LOCAL SERVICE", "Local Service Account"))
+			.put("S-1-5-20", new WellKnownSidInfo(true, "S-1-5", NTAUTHORITY_REALM_NAME, "NETWORK SERVICE", "Network Service Account"))
 			.build();
 		
-	private static final Map<String, String> SPECIAL_SID_PREFIXES_MAP = ImmutableMap.<String, String>builder() 
-			.put("S-1-5-80", "Service Virtual Account")
-			.put("S-1-5-82", "IIS AppPool Virtual Account")
-			.put("S-1-5-83", "Virtual Machine Virtual Account")
-			.put("S-1-5-90", "Window Manager Virtual Account")
-			.put("S-1-5-94", "WinRM Virtual accountt")
-			.put("S-1-5-96", "Font Driver Host Virtual Account")
+
+	// These SID prefixes indicate well known windows accounts.
+	//  - We can fill in the login names for these SID, as well as account user description.
+	private static final Map<String, WellKnownSidInfo> SPECIAL_SID_PREFIXES_MAP = ImmutableMap.<String, WellKnownSidInfo>builder() 
+			.put("S-1-5-80", new WellKnownSidInfo(false, "S-1-5-80", "NT Service", "", "NT Service Virtual Account"))
+			.put("S-1-5-82", new WellKnownSidInfo(false, "S-1-5-82", "IIS AppPool", "", "IIS AppPool Virtual Account"))
+			.put("S-1-5-83", new WellKnownSidInfo(false, "S-1-5-83", "NT Virtual Machine", "", "Virtual Machine Virtual Account") )
+			.put("S-1-5-90", new WellKnownSidInfo(false, "S-1-5-90", "Window Manager", "", "Windows Manager Virtual Account"))
+			.put("S-1-5-94", new WellKnownSidInfo(false, "S-1-5-94", "Windows Remoting Virtual Users", "", "Windows Remoting Virtual Account"))
+			.put("S-1-5-96",  new WellKnownSidInfo(false, "S-1-5-96", "Font Driver Host", "", "Font Driver Host Virtual Account"))
 			.build();
 			
 	
 	// Looks for security identifier prefixes of the form S-<number>-<number>-<number>
-	//More information on security identifier architecture can be found at: 
+	// More information on security identifier architecture can be found at: 
 	// https://docs.microsoft.com/en-us/windows/security/identity-protection/access-control/security-identifiers
-	private static final Pattern WINDOWS_SPECIAL_ACCOUNT_REGEX = Pattern.compile("^\\s*S\\-\\d*\\-\\d*\\-(\\d*)");
+	// A number of accounts in the range S-1-5-80-* to S-1-5-111-* are special. 
+	private static final Pattern WINDOWS_SPECIAL_ACCOUNT_PREFIX_REGEX = Pattern.compile("^\\s*S\\-1\\-5\\-(\\d*)\\-");
```

Comment:
- Do we want this to be matching if the SID string has spaces in the front?
- This seems to only match if the S is capital. We should change it to [sS] to account for both cases

Reply:
Addressed.